gsad/config/gsad.service.in

```diff
@@ -1,16 +1,19 @@
 [Unit]
-Description=Greenbone Security Assistant
+Description=Greenbone Security Assistant daemon (gsad)
```

Comment:
See the recent discussion about this in https://github.com/greenbone/gvmd/pull/1240/files#r467782260

We should find a consensus here to use a consistent Description for all systemd units